SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/LayoutInfoViewModel.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ *
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.duosamples.about
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class LayoutInfoViewModel : ViewModel() {
+    var isDualMode = MutableLiveData(false)
```

Comment:
To be honest, I didn't find the point from where this isDualMode var is updated.